eventbroadcaster/notifiersink.go

```diff
@@ -0,0 +1,97 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2021 Datadog, Inc.
+
+package eventbroadcaster
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/DataDog/chaos-controller/api/v1beta1"
+	"github.com/DataDog/chaos-controller/eventnotifier"
+
+	ctrl "sigs.k8s.io/controller-runtime"
+
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/client-go/tools/record"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+type NotifierSink struct {
+	client   client.Client
+	notifier eventnotifier.Notifier
+}
+
+// RegisterNotifierSinks builds notifiers sinks and registers them on the given broadcaster
+func RegisterNotifierSinks(mgr ctrl.Manager, broadcaster record.EventBroadcaster, notifiersConfig eventnotifier.NotifiersConfig) (err error) {
+	err = nil
+
+	client := mgr.GetClient()
+
+	notifiers, err := eventnotifier.GetNotifiers(notifiersConfig)
+
+	for _, notifier := range notifiers {
+		broadcaster.StartRecordingToSink(&NotifierSink{client: client, notifier: notifier})
+	}
+
+	return
+}
+
+func (s *NotifierSink) Create(event *corev1.Event) (*corev1.Event, error) {
+	dis, err := s.getDisruption(event)
+
+	if err != nil {
+		return event, nil
+	}
+
+	if err = s.parseEventToNotifier(event, dis); err != nil {
+		fmt.Println(err)
```

Comment:
Expected? It'll print an error on standard output with no formatting while I would expect more something handled by a logger here.